src/app/core/disco/disco.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core'
+import { Observable } from 'rxjs'
+import { environment } from '../../../environments/environment'
+import { HttpClient } from '@angular/common/http'
+import { Disco } from '../../types/disco.endpoint'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DiscoService {
+  constructor(private http: HttpClient) {}
+
+  getDiscoFeed(): Observable<any> {
```

Comment:
This getDiscoFeed should be of type Observable<Disco>

Reply:
Ok. I will fix it.